devicetypes/smartthings/motion-detector.src/motion-detector.groovy

```diff
@@ -44,6 +43,11 @@ metadata {
 
 def installed() {
 	initialize()
+	if(isAuroraMotionSensor51AU()) {
+		// Aurora Smart PIR Sensor doesn't report when there is no motion during pairing process
+		// reports are sent only if there is motion detected, so fake event is needed here
+		sendEvent(name: "motion", value: "inactive", descriptionText: "${device.displayName} - initially no motion" )
```

Comment:
You don't need the `descriptionText` but you should set `displayed: false`

Reply:
ok, fixed